dspace-api/src/main/java/org/dspace/app/util/CacheSnooper.java

```diff
@@ -0,0 +1,49 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+
+package org.dspace.app.util;
+
+import net.sf.ehcache.Cache;
+import net.sf.ehcache.CacheManager;
+import org.dspace.core.Context;
+import org.dspace.servicemanager.DSpaceKernelImpl;
+import org.dspace.servicemanager.DSpaceKernelInit;
+import org.dspace.services.CachingService;
+
+/**
+ * List all EhCache CacheManager and Cache instances.
```

Comment:
I would recommend we enhance the comments here to detail how to call this class, and also likely note that it's for debugging/testing purposes *only*.  Otherwise, the goal of this class may not be immediately clear, since it is obviously not used anywhere in the code (and IDEs my continually point out that it's not used)

Reply:
Done!